sale_layout_category_hide_detail/readme/USAGE.rst

```diff
@@ -0,0 +1,45 @@
+To use this module, you need to:
+
+#. Go to *Sales -> Orders -> Quotations* and create a new *Quotation*.
+#. In *Order lines* tab add a section line.
+#. Add some product lines.
+#. Add another section line, but this time, click on the *plus-circle*
+   icon |plus-circle-icon| on your right. That icon will be replaced by
+   *minus-circle* icon |minus-circle-icon|. That mean *Show subtotal* field is
+   set to False.
+#. Add some product lines.
+#. Add another section line and click on the *eye* icon |eye-icon| on your
+   right. That icon will be replaced by *eye-slash* icon |eye-slash-icon|.
+   That mean *Show details* field is set to False.
+#. Add some product lines.
+#. Print a *Quotation / Order* report for this quotation.
+
+After following the steps described above, in the report you will see the
+following:
+
+  * The first 'line section' and its product order lines will be shown in
+    a normal way.
+  * The second 'line section' and its product order lines will be shown in
+    a normal way, but the subtotal won't be shown. That is because in this
+    section line *Show subtotal* field was set to False.
+  * The third 'line section' will show the name on the left and the
+    subtotal on the right. Besides, its product order lines won't be shown.
+    That is because in this line *Show details* field was set to False.
```

Comment:
The flags on the right hand side of the sale order line allow you to show or hide details, subtotal, line amount and section subtotal.

These are the possible combinations: 

![image](https://user-images.githubusercontent.com/55489873/190119534-49e10e89-c77e-444c-974c-6c35bf32c803.png)


And these are the results in report: 

![image](https://user-images.githubusercontent.com/55489873/190119587-2e030cfd-02a2-42d1-8b4f-f6a7fe3aeeff.png)

Reply:
Would you replace all the previous content for the content you wrote here?

Reply:
Done, I'm not completely sure that images are correcly added. Could you check?

Reply:
The description is ok, let me know when I can test functionally again

Reply:
It's ready for functional review